rpc/core/status.go

```diff
@@ -71,6 +71,8 @@ func (env *Environment) Status(ctx *rpctypes.Context) (*ctypes.ResultStatus, err
 			EarliestBlockTime:   time.Unix(0, earliestBlockTimeNano),
 			MaxPeerBlockHeight:  env.FastSyncReactor.GetMaxPeerBlockHeight(),
 			CatchingUp:          env.ConsensusReactor.WaitSync(),
+			TotalSyncedTime:     env.FastSyncReactor.GetTotalSyncedTime(),
+			RemainingTime:       env.FastSyncReactor.GetRemainingSyncTime(),
```

Comment:
what happens to nodes that: 

- do not have statesync enabled? 
- have finished sysncing?
- have state sync enabled but switched to consensus directly after starting up?

Reply:
> what happens to nodes that:
> 
> * do not have statesync enabled?
Will tackle this case, return `0`
> * have finished sysncing?
CatchingUp will show the status of the syncing, I will tackle this case, return `0` too.
> * have state sync enabled but switched to consensus directly after starting up?
Same above. if reactor.WaitSync is false, these two fields will be `0`

Reply:
I worry that nodes that don't have fastsync enabled the value of `env.FastSyncReactor` could (and should?) be nil, which means this call will panic.

Reply:
The blockchain reactor will always be created if  I understand correctly
https://github.com/tendermint/tendermint/blob/b317df4ba5db721e4b3dd976e4fbebb36969857d/node/node.go#L291
and
https://github.com/tendermint/tendermint/blob/b317df4ba5db721e4b3dd976e4fbebb36969857d/node/node.go#L655